src/Symfony/Component/Notifier/Bridge/SmsBiuras/README.md

```diff
@@ -0,0 +1,26 @@
+SmsBiuras Notifier
+==================
+
+Provides [SmsBiuras](https://www.smsbiuras.lt) integration for Symfony Notifier.
+
+DSN example
+-----------
+
+```
+SMSBIURAS_DSN=smsbiuras://UID:TOKEN@default?from=FROM&test=0
+```
+
+where:
+- `UID` is your client code
+- `TOKEN` is your SmsBiuras token
```

Comment:
Please rename TOKEN to API_KEY and $token to $apiKey because they name it the same:

![image](https://user-images.githubusercontent.com/995707/113544982-9d113000-95e9-11eb-911c-875d09e79b36.jpeg)